karapace/statsd.py

```diff
@@ -125,8 +125,8 @@ def _send(self, metric: str, metric_type: bytes, value: Any, tags: Optional[Dict
                 parts.insert(1, ",{}={}".format(tag, tag_value).encode("utf-8"))
 
             self._socket.sendto(b"".join(parts), self._dest_addr)
-        except Exception as ex:  # pylint: disable=broad-except
-            self.log.error("Unexpected exception in statsd send: %s: %s", ex.__class__.__name__, ex)
```

Comment:
Why not log the `Exception`?

Reply:
the change is from `error` to `exception`. That is called `exception` because the exception is included.